src/bin/common/parse_command.ml

```diff
@@ -462,13 +462,13 @@ let mk_theory_opt () no_contracongru
     no_fm no_nla no_tcp no_theory restricted tighten_vars
     _use_fpa (theories)
   =
-  set_no_ac (not (Lists.mem Theories.equal Theories.AC theories));
+  set_no_ac (not (My_list.mem Theories.equal Theories.AC theories));
```

Comment:
Nit: I think we could get rid of `My_list.mem` everywhere with `List.exists (Theories.equal Theories.AC) theories`. Not sure we want to.

Reply:
Sure